.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+yarn.lock
```

Comment:
What's your justification for ignoring `yarn.lock`?

Reply:
Oh, I can remove this commit. It's simply that installing this in dev will create either a `yarn.lock`, or a `package.json.lock`. These aren't typically committed for libraries.